common/solar_system_initializers/hfe_initializer.txt

```diff
@@ -2,11 +2,14 @@
 # egypt
 hfe_initializer_egypt_main = {
 	prevent_anomalies = yes
+	usage = fallen_empire_init
```

Comment:
No

Reply:
You didn't read the whole thing. The Initializer DOES NOT RANDOMLY SPAWN. It has a spawn chance of ZERO. It is ONLY spawned by your Event.

Reply:
And whats point?

Reply:
Categorization and for referencing, of course!

Reply:
Make no sense

Reply:
I just explained why. If you categorize them, they need to be ensured not to spawn. Why are you reopening so many dead issues?

Reply:
I see no sense in such categorizing

Reply:
Categories are always good. It doesn't change anything except making it sorted in the game (and for searching purposes in the code).

Reply:
Do you remember than game spawned civics with 0 chance?

Reply:
This is not a Civic file.

Reply:
Who know what they gonna ruin next time.